tests/test_dice_interface/test_dice_genetic.py

```diff
@@ -46,6 +46,7 @@ def _initiate_exp_object(self, genetic_binary_classification_exp_object):
     @pytest.mark.parametrize("desired_class, total_CFs", [(7, 3)])
     def test_no_cfs(self, desired_class, sample_custom_query_1, total_CFs):
         with pytest.raises(UserConfigValidationException):
+            self.exp.data_interface.set_continuous_feature_indexes(query_instance=sample_custom_query_1)
             self.exp._generate_counterfactuals(query_instance=sample_custom_query_1, total_CFs=total_CFs,
```

Comment:
Why do you need to call this function from the test? Ideally, you should call generate_counterfactuals from here instead of _generate_counterfactuals()

Reply:
Thanks for the comment! I've restructured the tests to call generate_counterfactuals instead of _generate_counterfactuals().

Reply:
Thanks @soundarya98. The tests are looking cleaner and better.